builder/steps/gen-dockerfile/build.sh

```diff
@@ -28,7 +28,7 @@ if [ -z "${BUILDER_NAMESPACE}" -o -z "${BUILDER_TAG}" -o -z "${UPLOAD_TO_STAGING
   exit 1
 fi
 
-UNTAGGED_BUILDER_NAME=${BUILDER_NAMESPACE}/${RUNTIME_NAME}
+UNTAGGED_BUILDER_NAME=${BUILDER_NAMESPACE}/nodejs/${RUNTIME_NAME}
```

Comment:
It's good to make sure the syntax for the name is valid because the ruby team is using the same naming convention ([ruby.yaml.in](https://github.com/GoogleCloudPlatform/ruby-docker/blob/master/builder/ruby.yaml.in)).